src/Form/UserSubscriptionsForm.php

```diff
@@ -77,6 +81,23 @@ public function getFormId() {
    */
   public function buildForm(array $form, FormStateInterface $form_state, UserInterface $user = NULL) {
     $this->account = $user;
+    $subscriptions_config = $this->configFactory->get(SettingsForm::CONFIG_NAME);
+
+    if (!empty($subscriptions_config->get('subscriptions_page_text'))) {
```

Comment:
Instead of this, let's put it in a template.

Reply:
Added